Libplanet.Explorer.UnitTests/GraphTypes/BlockTypeTest.cs

```diff
@@ -45,13 +46,16 @@ public async void Query()
             Assert.Null(result.Errors);
             Assert.Equal(
                 block.Index,
-                Convert.ToInt64(((Dictionary<string, object>) result.Data)["index"]));
+                ((Dictionary<string, object>) result.Data)["index"]);
```

Comment:
How about introducing a variable to replace these repetitive `((Dictionary<string, object>) result.Data)`?

Reply:
I applied your suggestion in 258ff13.